### PR TITLE
Fix input icon positions in safari

### PR DESCRIPTION
**Pull request by ryandialpad**
## Description
Added `top: 0` to input icon class to correctly position the input icon in Safari

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [x] Update, remove, or extend all affected documentation.
 - [x] Provide a description what is being changed, extended, or removed.
 - [x] Link to any issue(s) this request is resolving.
 - [x] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
 - [x] Request a review from Joshua Hynes, Ted Goas, or David Becher.

#### Chrome

Screen Shot 2021-09-28 at 3 46 37 PM

#### Safari

Screen Shot 2021-09-28 at 3 46 52 PM

#### Chrome
Screen Shot 2021-09-28 at 3 53 36 PM

#### Safari
Screen Shot 2021-09-28 at 3 53 18 PM

### CODE REVIEW

**Comment by ryandialpad** [comment]
Testing locally in Dialtone Vue

**Comment by ryandialpad** [comment]
Dialtone Vue local testing complete

File: docs/components/input.html [hunk 1]
```diff
@@ -236,20 +236,20 @@
...
                   <input class="d-input d-input-icon--left" id="Dialtone--InputExample--IconLeft" type="text" placeholder="Placeholder" />
                 </div>
               </div>
-              <div class="d-w100p d-ps-relative">
+              <div class="d-w100p">
                 <div>
                   <label class="d-label" for="Dialtone--InputExample--IconRight">Label</label>
                 </div>
-                <div>
+                <div class="d-ps-relative">
```
Comment (dbecherdialpad) [inline docs/components/input.html:252]: @ryandialpad @braddialpad as discussed in the other PR do you think this should be an "official" dialtone class? `.d-input__wrapper` or something?

**Review by dbecherdialpad — APPROVED** [review]

File: docs/components/input.html (hunk 1 above)
Comment (braddialpad) [reply docs/components/input.html:252]: Yes I think so, makes it easier to identify

**Review by braddialpad — APPROVED** [review]

File: docs/components/input.html (hunk 1 above)
Comment (ryandialpad) [reply docs/components/input.html:252]: I will create a follow up PR for that

**Pull request merged**